showallthehooks.php

```diff
@@ -8,6 +8,11 @@
 
 require_once 'showallthehooks.hooks.php';
 
+// if WordPress, add hook callbacks file
+if (function_exists('add_action')) {
```

Comment:
I would probably test CiviCRM's `CRM_Core_Config::singleton()->userSystem->is_wordpress` to avoid this being triggered by some other custom code that happened to define `add_action()`.